discovery-medicine.csl

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Discovery Medicine</title>
+    <title-short>DM</title-short>
+    <id>http://www.zotero.org/styles/discovery-medicine</id>
+    <link href="http://www.zotero.org/styles/discovery-medicine" rel="self"/>
+    <link href="http://www.zotero.org/styles/brain" rel="template"/>
+    <link href="http://www.discoverymedicine.com/about/information-for-authors-of-discovery-medicine/" rel="documentation"/>
+    <author>
+      <name>O'Brien Patrick, PhD</name>
```

Comment:
@damnation333, do you care about the order here? I've been changing a few `<name>O'Brien Patrick, PhD</name>` to `<name>Patrick O'Brien, PhD</name>` in recent submissions.